tests/test_qdrant_client.py

```diff
@@ -69,7 +70,7 @@ def test_qdrant_client_integration(prefer_grpc):
         vectors=vectors,
         payload=payload,
         ids=None,  # Let client auto-assign sequential ids
-        parallel=2
+        parallel=1
```

Comment:
Why this change? It would be nice to keep it to catch any strange behavior due to multithreading.

Reply:
but it's definitely a NIT.

Reply:
Actually, I changed this to debug something. Might be it would be better to improve error reporting in parallel uploading mode :writing_hand:

Reply:
yeah you're right --it would be better to test them in isolation 👍

Reply:
I found out that error was actually logged, but as a side-effect I also fixed the proper shutdown in case of exception